frontend/src/themes/components/radio.ts

```diff
@@ -23,6 +23,15 @@ const Radio = {
         color: "grey.300",
       },
     },
+    table: {
+      control: {
+        borderColor: "grey.300",
+        _checked: {
+          backgroundColor: "grey.300",
+          borderColor: "grey.300",
+        },
+      },
+    },
```

Comment:
I made this grey instead of blue like in the design, since if you make it blue, the selected radio becomes invisible on hover.